generative-ai/langchain-rag.ipynb

```diff
@@ -700,7 +868,7 @@
     "\n",
     "# Summary and next steps\n",
     "\n",
-    "In this tutorial, you created a LangChain RAG system in Python with watsonx. You fetched 27 articles from https://www.ibm.com to create a vector store as context for an LLM to answer questions about the Think 2024 conference.\n",
+    "In this tutorial, you created a simple LangChain RAG workflow in Python with watsonx. You fetched 26 articles from https://www.ibm.com to create a vector store as context for an LLM to answer questions about IBM offerings and technologies.\n",
```

Comment:
Using `[ibm.com](https://www.ibm.com)` could be cleaner than https://www.ibm.com

Reply:
updated